### PR TITLE
Backported to 1.20.1 Fabric & Forge

### PR DESCRIPTION
**Pull request by marlester-dev**
```
Downgraded to 1.20.1.
Thoroughly tested, the mod is working. The config is working. The biomes are working. The trees are working.
The basis for this was 1.20.4-multi.
With this change the mod will be available for 1.20.1 Fabric, which very many players and modpack creators want.
```

### CODE REVIEW

**Comment by marlester-dev** [comment]
I want you to create a 1.20.1-multi branch based off 1.20.4-multi so I can merge it there

**Comment by marlester-dev** [comment]
I made this a draft, because I want to try to do some stuff here, I will finish this soon.

File: forge/src/main/resources/META-INF/mods.toml [hunk 1]
```diff
@@ -22,7 +22,7 @@ logoFile="${mod_id}_logo.png"
 [[dependencies.${mod_id}]]
     modId="glitchcore"
     mandatory=true
-    versionRange="[${glitchcore_version},)"
+    versionRange="[0.0.0,)"
```
Comment (Adubbz) [inline forge/src/main/resources/META-INF/mods.toml]: This needs to be changed back.

File: fabric/src/main/resources/fabric.mod.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
     "fabric": "*",
     "minecraft": "${minecraft_version}",
     "java": ">=17",
-    "glitchcore": ">=${glitchcore_version}",
+    "glitchcore": "*",
```
Comment (Adubbz) [inline fabric/src/main/resources/fabric.mod.json]: Revert this

**Pull request merged**